[rospy.client][INFO] 2017-06-05 19:16:40,995: init_node, name[/cartesian_control_29431_1496715400957], pid[29431]
[xmlrpc][INFO] 2017-06-05 19:16:40,995: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-06-05 19:16:40,999: Started XML-RPC server [http://ip-172-31-19-100:57683/]
[rospy.init][INFO] 2017-06-05 19:16:40,999: ROS Slave URI: [http://ip-172-31-19-100:57683/]
[rospy.impl.masterslave][INFO] 2017-06-05 19:16:41,003: _ready: http://ip-172-31-19-100:57683/
[xmlrpc][INFO] 2017-06-05 19:16:41,012: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-06-05 19:16:41,012: Registering with master node http://localhost:57311
[rospy.init][INFO] 2017-06-05 19:16:41,108: registered with master
[rospy.rosout][INFO] 2017-06-05 19:16:41,112: initializing /rosout core topic
[rospy.rosout][INFO] 2017-06-05 19:16:41,122: connected to core topic /rosout
[rospy.simtime][INFO] 2017-06-05 19:16:41,127: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-06-05 19:16:41,196: topic[/joint_states] adding connection to [http://ip-172-31-19-100:46286/], count 0
[rospy.internal][INFO] 2017-06-05 19:16:41,346: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-06-05 19:16:41,356: topic[/joint_command] adding connection to [/robot_sim], count 0
[rospy.internal][INFO] 2017-06-05 19:16:41,360: topic[/control_markers/update] adding connection to [/interactive_marker_proxy], count 0
[rospy.internal][INFO] 2017-06-05 19:16:41,559: topic[/control_markers/update_full] adding connection to [/interactive_marker_proxy], count 0
[rospy.internal][INFO] 2017-06-05 19:16:41,616: topic[/control_markers/update_full] removing connection to /interactive_marker_proxy
[rospy.internal][INFO] 2017-06-05 19:17:03,617: topic[/control_markers/feedback] adding connection to [http://ip-172-31-19-100:59243/], count 0
[rospy.internal][INFO] 2017-06-05 19:17:44,380: topic[/redundancy_command] adding connection to [/cartesian_control_30748_1496715464139], count 0
[rospy.internal][INFO] 2017-06-05 19:17:44,387: topic[/cartesian_command] adding connection to [/cartesian_control_30748_1496715464139], count 0
[rospy.internal][ERROR] 2017-06-05 19:18:19,708: Unable to initiate TCP/IP socket to ip-172-31-19-100:48421 (http://ip-172-31-19-100:59243/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: Internal error processing topic [/control_markers/feedback]

[rospy.internal][ERROR] 2017-06-05 19:18:20,711: Unable to initiate TCP/IP socket to ip-172-31-19-100:48421 (http://ip-172-31-19-100:59243/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: [/rosbridge_websocket] is not a publisher of [/control_markers/feedback]. Topics are [['/rosout', 'rosgraph_msgs/Log']]

